admin/modules/user/mass_mail.php

```diff
@@ -675,6 +691,7 @@ function insertText(value, textarea)
 		{
 			$page->output_inline_error($errors);
 			$input = $mybb->input;
+			$delivery_type_checked[$input['delivery_type']] = " checked=\"checked\"";
```

Comment:
I think we need to check if `$input['delivery_type']` is set. Maybe just `$mybb->get_input('delivery_type')`